/**:
  ros__parameters:
    roughness_threshold: 0.1
    slope_threshold: 0.1
    distance_threshold: 0.1